### DISCUSSION TITLE
"Updating Workspace Database"

### DISCUSSION THREAD

**Discussion by Jeremiah-SC** [TabularEditor/TabularEditor3#512] [🙋♀️TE3 Q&A] [answered]
I have a tabular model I've saved ("save to folder") in my git folder locally after pulling from the live model on PBI Premium.  This morning, when I opened up Tabular editor and tried to open my model using the "open from folder" option, I get a popup that spins forever (been waiting 30 minutes) which says "updating workspace database".  

I have not tried to deploy anything, but am just opening the locally stored model from TE3.

What is it doing?  Why does it take this long?

**Comment by otykier** [comment, marked as answer]
Hi @Jeremiah-SC 
If you've previously connected to a workspace database with this model, then TE3 will try to connect to and update the model when loading the metadata. Normally, this should not take more than 1 or 2 minutes at most, but there could be circumstances where the Power BI service is slower - [enabling Large Datasets in your Power BI workspace](https://docs.microsoft.com/en-us/power-bi/enterprise/service-premium-large-models#enable-large-datasets) usually solves this problem.

If you do not wish to open the metadata while connected to a workspace database (thus, completely skipping the 'updating workspace database' step), simply delete the .tmuo file next to the database.json file, before loading the model in Tabular Editor. This causes the tool to prompt you whether you want to use a workspace database or not.

**Reply by Jeremiah-SC** [reply]
Thanks @otykier !  Do you know if there's a setting that prevents it from creating .tmuo files generally?

**Reply by otykier** [reply]
You can’t prevent the .tmuo file from being created (make sure to add this extension to the .gitignore file if you’re using git). However, if you choose the “Don’t ask again”-option when loading the model, TE3 will not use a workspace database and will not ask you again the next time that model is loaded (if you don’t get the prompt, simply delete the .tmuo file and load the model again)
